src/main/flight/imu.c

```diff
@@ -811,3 +811,8 @@ bool isUpright(void)
     return true;
 #endif
 }
+
+float isupsidedown(void){
+    imuComputeRotationMatrix();
+    return rMat[2][2];
+}
```

Comment:
isUpsideDown suggest that it returns a boolean. Not sure what else to call it though haha